launch/test/counter.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import argparse
+import signal
+import sys
+import time
+
+
+def main(argv=sys.argv[1:]):
+    parser = argparse.ArgumentParser(
+        description=(
+            'Simple program outputting a counter. '
+            'Even values go to STDERR, odd values go to STDOUT.'))
```

Comment:
Not needed, but I've seen too many times that a multiline like this causes some breakage when moving variables around or reformatting. In this case it's fine because of the parenthesis, though.